src/version.h

```diff
@@ -9,7 +9,7 @@
  * network protocol versioning
  */
 
-static const int PROTOCOL_VERSION = 70010;
+static const int PROTOCOL_VERSION = 250000;
```

Comment:
I had previously thought that the version here was doubling as the block at which clients using the old code were disconnected.